whylog/constraints/__init__.py

```diff
@@ -126,12 +131,32 @@ def verify(self, group_contents, param_dict):
 
         pass
 
+    def _convert_params_values(self):
+        problems = []
+        for param_name, converter_type in six.iteritems(self.PARAMS):
+            param_val = self.params.get(param_name)
+            if param_val is not None:
```

Comment:
It is OK, because PARAMS has also optional params that could not exist in self.params